src/vivarium/framework/results/stratification.py

```diff
@@ -1,23 +1,23 @@
-# mypy: ignore-errors
 """
 ===============
 Stratifications
 ===============
 
 """
+from __future__ import annotations
 
-from dataclasses import dataclass
-from typing import Callable, List, Optional, Union
+from typing import Any, Callable
 
 import pandas as pd
 from pandas.api.types import CategoricalDtype
 
-from vivarium.types import ScalarValue
-
 STRATIFICATION_COLUMN_SUFFIX: str = "mapped_values"
 
 
-@dataclass
+VectorMapper = Callable[[pd.DataFrame], pd.Series]  # type: ignore [type-arg]
+ScalarMapper = Callable[[pd.Series], str]  # type: ignore [type-arg]
```

Comment:
Why are you ignoring type-arg here? Why not `pd.Series[Any]`?

Reply:
from the types.py:
`    # TODO: Parameterizing pandas objects fails below python 3.12`
(when using TypeAliases, specifically, I assume)

Reply:
Oh right, I forgot about that. We should have a comment about this?